source/agora/registry/API.d

```diff
@@ -16,6 +16,7 @@ module agora.registry.API;
 import agora.common.Types;
 import agora.crypto.Hash;
 import agora.crypto.Key;
+import agora.crypto.Schnorr: Signature;
```

Comment:
Is this used ?

Reply:
https://github.com/bpfkorea/agora/blob/1efa307d45090915d40ba52ef124affafd398a45/source/agora/registry/API.d#L44